Lib/test/test_ssl.py

```diff
@@ -1824,6 +1824,14 @@ def test_bad_server_hostname(self):
             ctx.wrap_bio(ssl.MemoryBIO(), ssl.MemoryBIO(),
                          server_hostname="example.org\x00evil.com")
 
+    def test_gc_support(self):
```

Comment:
Please add a comment mentioning bpo-44252 to explain why you added this test. usually, we don't test the GC protocol explicitly on types.

Reply:
Ok, but I don't think is bad to add a test that checks that we have gc support for a given type properly

Reply:
I am actually going to remove the test